src/TraceEvent/ETWTraceEventSource.cs

```diff
@@ -161,6 +161,12 @@ public void ReprocessEvent(TraceEvent ev)
         /// The name of the session that generated the data. 
         /// </summary>
         public string SessionName { get { return logFiles[0].LoggerName; } }
+
+        /// <summary>
+        /// Resolution of the hardware timer, in units of 100 nanoseconds.
+        /// </summary>
+        public ulong TimerResolution { get { return logFiles[0].LogfileHeader.TimerResolution; } }
```

Comment:
The value in the header is of type `uint` is there a reason that it should be `ulong` here?

Reply:
My mistake. Addressed.